<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropTestParamsDialog</class>
 <widget class="QDialog" name="DropTestParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры дроп теста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblFinishKind">
        <property name="text">
         <string>Завершение теста</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFinishKind"/>
      </item>
      <item>
       <widget class="QFrame" name="frJumpsCount">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblJumpsCount">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Количество прыжков</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edJumpsCount">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frTestTime">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblTestTime">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Время, сек</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edTestTime">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFallHeight">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblFallHeight">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Высота спрыгивания, м</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="edFallHeight">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="btnDefault_2">
        <property name="text">
         <string>Сброс к настройкам по умолчанию...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK_2">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel_2">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbFinishKind</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DropTestParamsDialog</receiver>
   <slot>on_finishKindChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK_2</sender>
   <signal>clicked()</signal>
   <receiver>DropTestParamsDialog</receiver>
   <slot>on_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel_2</sender>
   <signal>clicked()</signal>
   <receiver>DropTestParamsDialog</receiver>
   <slot>on_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDefault_2</sender>
   <signal>clicked()</signal>
   <receiver>DropTestParamsDialog</receiver>
   <slot>on_default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_finishKindChanged(int)</slot>
  <slot>on_ok()</slot>
  <slot>on_cancel()</slot>
  <slot>on_default()</slot>
 </slots>
</ui>
